Packs/ExodusIntelligence/IndicatorFields/indicatorfield-cpes.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "indicator_cpes",
+    "version": -1,
+    "modified": "2022-04-29T11:08:49.074455322-04:00",
+    "propagationLabels": [
+        "all"
+    ],
+    "name": "CPES",
```

Comment:
Why are there 3 different fields for CPEs parts? what is going to be in this one?

Reply:
What are the other parts you are referring to?